src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentColumn.jsx

```diff
@@ -38,7 +38,7 @@ export default function AppointmentColumn({
   first,
   last,
   padding,
-  size,
+  size = 1,
```

Comment:
@JunTaoLuo - The class name on the <div> is 'undefined' when not specified.

Reply:
From my understanding, this should be a required property that must be set when using the component. At least this is what I see in AppointmentColumnLayout where this component is used. As such we should also set the value of size in the tests instead of adding a default value in the source. We can also consider marking this property as required in the `propTypes`.